rules/falco_rules.yaml

```diff
@@ -2741,7 +2741,7 @@
     command=%proc.cmdline pid=%proc.pid container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority:
     NOTICE
-  tags: [maturity_sandbox, host, container, process, users, mitre_persistence, T1548.001]
+  tags: [maturity_incubating, host, container, process, users, mitre_persistence, T1548.001]
```

Comment:
"Set Setuid or Setgid bit" relevant for broader attack category of privilege escalations.